bullet_train-fields/app/controllers/concerns/fields/date_and_time_support.rb

```diff
@@ -2,6 +2,12 @@ module Fields::DateAndTimeSupport
   extend ActiveSupport::Concern
 
   def assign_date_and_time(strong_params, attribute)
```

Comment:
I was able to reproduce the error you mentioned, we'll probably have to use another format for this line since the attribute is an iso8601 datetime now:
```ruby
strong_params[attribute] = time_zone.strptime(strong_params[attribute], t("global.formats.date_and_time"))
```

Still looking into it though, I'll try to see what option works best for us here.

Reply:
We could potentially parse the field to a ISO8601 `DateTime` object in and use this format instead.

```ruby
time_zone.strptime(DateTime.parse(strong_params[attribute]).iso8601, '%Y-%m-%dT%H:%S:%L%z')
```

Reply:
We might be able to do something like this until they update their strings like the deprecation your wrote tells them to.

```ruby
iso8601_format = "%Y-%m-%dT%H:%S:%L"
begin
  strong_params[attribute] = time_zone.strptime(strong_params[attribute], iso8601_format)
rescue ArgumentError => e
  # If developers are on >= 1.2.26, the format we get is different.
  iso8601_date = DateTime.strptime(strong_params[attribute], t("global.formats.date_and_time")).iso8601
  strong_params[attribute] = time_zone.strptime(iso8601_date, iso8601_format)
end
```